app/src/androidTest/java/org/simple/clinic/drugs/sync/PrescriptionSyncAndroidTest.kt

```diff
@@ -36,8 +41,26 @@ class PrescriptionSyncAndroidTest : BaseSyncCoordinatorAndroidTest<PrescribedDru
   @Inject
   lateinit var testData: TestData
 
+  @Inject
+  lateinit var userSession: UserSession
+
+  @Inject
+  lateinit var facilityRepository: FacilityRepository
+
+  val user: User
+    get() = userSession.loggedInUserImmediate()!!
+
+  val currentFacilityUuid: UUID
+    get() = facilityRepository.currentFacilityUuid(user)!!
+
+  val authenticationRule = AuthenticationRule()
+
+  val registerPatientRule = RegisterPatientRule(UUID.randomUUID())
+
   @get:Rule
-  val authenticationRule = AuthenticationRule(registerPatientWithUuid = UUID.randomUUID())
+  val ruleChain = RuleChain
```

Comment:
Apparently, JUnit doesn't support multiple `@Rule` annotations in a single test. It only executes one of them. If you need to apply multiple rules, you need to use [`RuleChain`](https://junit.org/junit4/javadoc/4.12/org/junit/rules/RuleChain.html).